src/disco/gui/fd_gui_printf.c

```diff
@@ -60,12 +60,23 @@ jsonp_long( fd_gui_t *   gui,
   else                   fd_http_server_printf( gui->http, "%ld,", value );
 }
 
+#define JSONP_DOUBLE_N(n)                                                                     \
+FD_FN_UNUSED static void                                                                      \
+jsonp_double_##n(fd_gui_t *   gui,                                                            \
```

Comment:
Why? Just print the whole thing at full precision, the frontend can round

Reply:
Actually I don't even know why we are printing skip rate here. Just print two ulongs, `slots_processed`, `slots_skipped`